KalturaWowzaServer/Installation.md

```diff
@@ -300,7 +300,7 @@ media_servers.ini is optional and needed only for custom configurations.
 
 
 **Edit @WOWZA_DIR@/conf/log4j.properties:**
- - Set `log4j.rootCategory` = `INFO`
+ - Set `log4j.rootCategory` = `INFO, stdout, serverAccess, kalturaAccess, serverError`
```

Comment:
actully this should be removed and  just copy the files /installation/configTemplates to it's correspdoning folders + rename the @@ properties